File: problem_forty_five.py
#area of triangle
import math
a=int(input("Enter value1:"))
b=int(input("Enter value2:"))
c=int(input("Enter value3:"))
def MyFunction(a,b,c):
    s= (a+b+c)/2
    area = math.sqrt(s*(s-a)*(s-b)*(s-c))
    return round(area,3)

print("Area =",MyFunction(a,b,c))

File: problem_forty_five.js
//area of triangle
var inp1=13
var inp2 =18
var inp3 = 15
function myFunction(a,b,c){
    var s = (a+b+c)/2
    var area = Math.sqrt(s*(s-a)*(s-b)*(s-c))
    return area.toFixed(3)
}
console.log("area =",myFunction(inp1,inp2,inp3))